NYPLCardCreator/Flow/UserCredentialsViewController.swift

```diff
@@ -69,7 +69,7 @@ final class UserCredentialsViewController: TableViewController {
   override func viewDidLoad() {
     super.viewDidLoad()
     
-    self.tableView.backgroundColor = UIColor.groupTableViewBackground
+    self.tableView.backgroundColor = NYPLColor.disabledFieldTextColor
```

Comment:
this is a background color but we are assigning a text color... should we maybe use the secondary background color? not sure.... in UserSummaryVC you used primaryBackgroundColor, which is also fine of course.

Reply:
Nice catch, thanks